torchtune/modules/loss/cross_entropy_loss.py

```diff
@@ -9,9 +9,9 @@
 from torch import nn
 from torch.distributed.tensor import DTensor
 
+import liger_kernel.ops.fused_linear_cross_entropy
```

Comment:
We should move this into the init of the model as they're an optional dependency